README.md

```diff
@@ -799,6 +803,78 @@ If you are using a feature flag with binary variants, the `StatusOverride` prope
 
 In the above example, the feature is enabled by the `AlwaysOn` filter. If the current user is in the calculated percentile range of 10 to 20, then the `On` variant is returned. Otherwise, the `Off` variant is returned and because `StatusOverride` is equal to `Disabled`, the feature will now be considered disabled.
 
+## Telemetry
+
+When a feature flag changed is deployed, it is often important to analyze its affect on an application. For example here are a few questions that may arise:

```

Comment:
```suggestion
When a feature flag change is deployed, it is often important to analyze its affect on an application. For example here are a few questions that may arise:
```

Reply:
Or "When a feature flag is changed and deployed"?

Reply:
effect? (affect is a verb)

Reply:
Updated.